ast-helper.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const R = require('ramda');
+
+// :: Node -> String
+const getName = R.ifElse(
+    R.propEq('type', 'MemberExpression'),
+    R.path(['property', 'name']),
+    R.prop('name')
+);
+
+// :: String -> Node -> Boolean
+const isName = name => R.either(
```

Comment:
I think `isName` is not very expressive, I would suggest something like `isRamdaMethod`.